packages/ember-metal/lib/computed.js

```diff
@@ -426,11 +426,11 @@ ComputedPropertyPrototype._set = function computedPropertySet(obj, keyName, valu
 };
 
 /* called before property is overridden */
-ComputedPropertyPrototype.teardown = function(obj, keyName) {
+ComputedPropertyPrototype.teardown = function(obj, keyName, _meta) {
   if (this._volatile) {
     return;
   }
-  let meta = metaFor(obj);
```

Comment:
another question though is, if teardown isn't public and the only place we call it we already provide meta, then maybe we don't need to do a meta lookup at all? What are your thoughts?